src/app/inventory/item-types.ts

```diff
@@ -152,6 +152,11 @@ export interface DimItem {
   /** Sometimes the API doesn't return socket info. This tells whether the item *should* have socket info but doesn't. */
   missingSockets: boolean;
 
+  /** The state of this item in the user's D2 Collection */
+  collectibleState: number;
```

Comment:
Would you mind changing this to the enum type exported by the API typings? I think it’s DestinyCollectibleState.